packages/requisitions/src/ResponseRequisition/api/api.ts

```diff
@@ -123,6 +125,25 @@ export const getResponseQueries = (sdk: Sdk, storeId: string) => ({
 
     throw new Error('Unable to update requisition');
   },
+  deleteLines: async (_: ResponseLineFragment[]) => {
+    // NOT YET IMPLEMENTED IN SERVER API
+
+    // const ids = responseLines.map(responseParser.toDeleteLine);
+    // const result = await sdk.deleteRequestLines({ ids, storeId });
+
+    // if (result.batchRequestRequisition.deleteRequestRequisitionLines) {
+    //   const failedLines =
+    //     result.batchRequestRequisition.deleteRequestRequisitionLines.filter(
+    //       line =>
+    //         line.response.__typename === 'DeleteRequestRequisitionLineError'
+    //     );
+    //   if (failedLines.length === 0) {
+    //     return result.batchRequestRequisition.deleteRequestRequisitionLines;
+    //   }
+    // }
+
+    throw new Error('Could not delete requisition lines!');
+  },
```

Comment:
Was part-way through adding this "Delete" action when I realised there is not yet any back-end API for `deleteResponseRequisitionLine` (only `deleteRequestRequisitionLine`). However I figured I may as well leave the boilerplate in as a placeholder (it's in a few places, basically a copy of the "Request" equivalent) until it's ready.